frontend/src/pages/NewRun.tsx

```diff
@@ -535,10 +569,9 @@ class NewRun extends Page<{}, NewRunState> {
   }
 
   private _start(): void {
-    const { pipelineFromRun, pipeline, usePipelineFromRun } = this.state;
```

Comment:
Curious why this change?

Reply:
Consistency mostly. We were pulling a few variables out of state like this, and not doing so for others within this same function, and it didn't seem like we were getting much for doing so